### PR TITLE
new issue templates using forms

### PR DESCRIPTION
**Pull request by zkat**
Summary of the changes (in less than 80 characters):

This updates the issue templates for this issue to use github forms!

### Preview

* [Feature](https://github.com/NuGet/NuGetGallery/blob/dev-zkat-new-issue-templates/.github/ISSUE_TEMPLATE/FEATURE.yml)
* [NuGet.org Issue](https://github.com/NuGet/NuGetGallery/blob/dev-zkat-new-issue-templates/.github/ISSUE_TEMPLATE/NUGETORG_ISSUE.yml)
* [Self-hosted Issue](https://github.com/NuGet/NuGetGallery/blob/dev-zkat-new-issue-templates/.github/ISSUE_TEMPLATE/SELF_HOSTED.yml)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/NUGETORG_ISSUE.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+       ### For issues regarding your NuGet.org account, please refer to [this guide](https://docs.microsoft.com/en-us/nuget/faqs/nuget-faq#nugetorg-account-management).
+  - type: dropdown
+    id: impact
+    attributes:
+      label: Impact
+      multiple: false
+      description: How does this issue impact you?
+      options:
+        - I'm unable to use this version
```
Comment (loic-sharma) [inline .github/ISSUE_TEMPLATE/NUGETORG_ISSUE.yml:23]: Consider this tweak given this is a website that isn't super clearly versioned as far as customers are concerned:

```suggestion
        - I'm unable to use NuGet.org
```

File: .github/ISSUE_TEMPLATE/NUGETORG_ISSUE.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  - type: textarea
+    id: repro
+    attributes:
+      label: Repro Steps
+      description: What steps can the NuGet team take to reproduce this issue?
+      placeholder: |
+        Steps to reproduce the behavior:
+        1. Using this git commit of NuGetGallery '...'
+        2. Run this code '....'
+        3. See error
```
Comment (loic-sharma) [inline .github/ISSUE_TEMPLATE/NUGETORG_ISSUE.yml:45]: This is a little more tailored to nuget.org website error reporting:

```suggestion
        1. On this URL '...'
        2. Perform this operation '...'
        3. See error
```

**Review by loic-sharma — APPROVED** [review]
Awesome, thanks for doing this!

File: .github/ISSUE_TEMPLATE/NUGETORG_ISSUE.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
```
Comment (joelverhagen) [inline .github/ISSUE_TEMPLATE/NUGETORG_ISSUE.yml:56]: Can you put screenshots in here? It looks like a textarea but maybe it just works.

Comment (zkat) [reply .github/ISSUE_TEMPLATE/NUGETORG_ISSUE.yml:56]: the textareas turn into regular GitHub text areas, with editor support and everything. You can try it out here: https://github.com/NuGet/Home/issues/new?assignees=&labels=Type%3ADCR%2CTriage%3AUntriaged&template=DCR.yml&title=%5BDCR%5D%3A+

**Review by joelverhagen — APPROVED** [review]
Looks great. Thanks for pushing this @zkat! Let's give this some time (24 hours maybe) so that other folks on the team and provide feedback if they have it.

**Review by zhhyu — APPROVED** [review]

**Comment by loic-sharma** [comment]
Thanks @zkat!

**Pull request merged**